mkt/developers/tasks.py

```diff
@@ -121,7 +122,7 @@ def run_validator(file_path, url=None):
     # uploaded file is on the local filesystem.
     temp_path = tempfile.mktemp()
     with open(temp_path, 'wb') as local_f:
-        with storage.open(file_path) as remote_f:
+        with private_storage.open(file_path) as remote_f:
```

Comment:
So, that means `File.file_path` and `FileUpload.path` are always private (since they are passed to `run_validator()`, but `File.signed_file_path` is public ? That makes sense to me, but I'm not sure it is at the moment.